modules/custom/az_marketing_cloud/az_marketing_cloud.module

```diff
@@ -0,0 +1,154 @@
+      'template' => 'views-view--az-marketing-cloud',
+      'base hook' => 'views_view',
+    ],
+    'views_view_unformatted__az_marketing_cloud' => [
+      'template' => 'views-view-unformatted--az-marketing-cloud',
+      'base hook' => 'views_view_unformatted',
+    ],
+
+  ];
+}
+
+/**
+ * Implements hook_preprocess_node().
+ */
+function az_marketing_cloud_preprocess_node__view__az_marketing_cloud(&$variables) {
+  $node = $variables['node'];
+  if ($node->getType() === 'az_news') {
+    $token_service = \Drupal::token();
+    $bubbleable_metadata = new BubbleableMetadata();
+    $az_canonical_url = $token_service->replace('[node:az-canonical-url]', ['node' => $node], [], $bubbleable_metadata);
+    $az_canonical_url_object = Url::fromUri($az_canonical_url, [
+      'language' => \Drupal::languageManager()->getCurrentLanguage(),
+    ]);
+    $az_canonical_absolute_url = $az_canonical_url_object->setAbsolute()->toString();
+    $variables['az_canonical_url'] = $az_canonical_absolute_url;
+    $variables['az_canonical_summary_link_title'] = $token_service->replace('[node:az-canonical-summary-link-title]', ['node' => $node], [], $bubbleable_metadata);
+    $media = $node->field_az_media_image->entity ?? NULL;
+    $file = $media->field_media_az_image->entity ?? NULL;
+    if (!empty($file)) {
+      $uri = $file->getFileUri();
+      $styles = [
+        'az_marketing_cloud_30_70_layout',
+        'az_marketing_cloud_50_50_layout',
+      ];
+      foreach ($styles as $style) {
+        $image_style = ImageStyle::load($style);
+        if (!empty($image_style)) {
+          $variables['images'][$style] = $image_style->buildUrl($uri);
+        }
+      }
+    }
+  }
+}
+
+/**
+ * Implements hook_theme_suggestions_HOOK_alter().
+ */
+function az_marketing_cloud_theme_suggestions_block_alter(array &$suggestions, array $variables) {
+  // If a view route context is found, add block suggestions for view context.
+  $view_id = \Drupal::routeMatch()->getParameter('view_id');
+  if (!empty($view_id)) {
+    $previous = $suggestions;
+    $view_id = strtr($view_id, '-', '_');
+    foreach ($previous as $p) {
+      $suggestions[] = $p . '__' . $view_id;
+    }
+  }
+}
+
+/**
+ * Implements hook_theme_suggestions_HOOK_alter().
+ */
+function az_marketing_cloud_theme_suggestions_region_alter(array &$suggestions, array $variables) {
+  $route_name = \Drupal::routeMatch()->getRouteName();
+  if ($route_name === 'view.az_marketing_cloud.50_50_layout') {
+    $suggestions[] = 'region__az_marketing_cloud';
+  }
+  if ($route_name === 'view.az_marketing_cloud.30_70_layout') {
+    $suggestions[] = 'region__az_marketing_cloud';
+  }
+  if ($route_name === 'view.az_marketing_cloud.text_layout') {
+    $suggestions[] = 'region__az_marketing_cloud';
+  }
+}
+
+/**
+ *
+ */
+function az_marketing_cloud_theme_suggestions_views_view_alter(array &$suggestions, array $variables) {
+  $route_name = \Drupal::routeMatch()->getRouteName();
```

Comment:
I would say do this, however the admin view is part of the display view, so this won't work
```suggestion
  $view_id = \Drupal::routeMatch()->getParameter('view_id');
```